scripts/build-backend.mjs

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/env node
 
+import {parseArgs} from 'util';
 import {$} from 'execa';
 import path from 'path';
 import fs from 'fs';
 import {downloadPrebuilds} from './download-prebuilds.mjs';
 
+const {
+  values: {prod},
```

Comment:
Where is this `prod` flag getting passed in?

Reply:
it's not getting used anywhere at the moment but when we want to enable prod-specific things in this script (e.g. backend minifying), we'd run this using `--prod` (open to renames)

Reply:
we may want to enable minifying by default (even in dev), which is what we do in mapeo mobile. mostly because it helps with app startup time